src/simple_cache_client.rs

```diff
@@ -307,7 +307,7 @@ impl SimpleCacheClient {
             key_id: kid.as_str().unwrap().to_owned(),
             key: res.key,
             expires_at: DateTime::<Utc>::from_utc(
-                NaiveDateTime::from_timestamp(res.expires_at as i64, 0),
+                NaiveDateTime::from_timestamp_opt(res.expires_at as i64, 0).unwrap(),
```

Comment:
Non-blocking: I'm going to cut a GitHub issue to capture this, but we are going through our private and public Rust repos to refactor use of `.unwrap()`. This is fine for now, but wanted to call out that we want to avoid generic use of `.unwrap()` in future code

Reply:
Edit: we already have this: https://github.com/momentohq/client-sdk-rust/issues/61